boardswarm-cli/src/main.rs

```diff
@@ -386,6 +386,54 @@ struct BmapWriteArgs {
     file: PathBuf,
 }
 
+#[derive(Clone, Debug)]
+enum VolumeArg {
+    Id(u64),
+    Name(String),
+}
+
+impl VolumeArg {
+    async fn volume(&self, mut client: Boardswarm) -> Result<Option<u64>, anyhow::Error> {
```

Comment:
Looking at this and all the other lookup functions; you'd probably want to just have one function that takes an ItemType to figure out what to search for

Reply:
done